.github/workflows/pr-lint-test-js.yml

```diff
@@ -13,12 +13,10 @@ jobs:
             - uses: actions/checkout@v3
 
             - name: Setup WooCommerce Monorepo
-              uses: ./.github/actions/setup-woocommerce-monorepo
-              with:
-                install-filters: 'woocommerce/client/admin...'
+              uses: ./.github/actions/setup-woocommerce-monorepo              
 
             - name: Lint
-              run: pnpm run lint --filter='woocommerce/client/admin...' --filter='!@woocommerce/e2e*' --filter='!@woocommerce/api' --color
+              run: pnpm run -r --filter='woocommerce/client/admin...' --filter='!@woocommerce/e2e*' --filter='!@woocommerce/api' --color lint 
```

Comment:
One thing I noticed as we moved to pnpm 7 is that the change of behaviour in `pnpm run` means that lint doesn't work as seamlessly with the turbo wrapped commands any more. I've added DEVELOPMENT.md updates for this but I think at some point maybe we should consider just putting `lint` back into turbo repo with cache: false because it makes every top level command you'd want to run work the exact same way.